a3p-integration/README.md

```diff
@@ -62,6 +64,8 @@ If the proposal is planned to be executed after the chain software upgrade, and
 
 For core eval proposals executing before the chain software upgrade, the `submission` should be checked in, since bundles built from newer software may not be compatible with older chains.
 
+For an example, see https://github.com/Agoric/agoric-sdk/pull/8907
```

Comment:
This might not be the best example since the generate-a3p-submission script changed after that. Could the one introducing `b:localchain` be a better example?

Reply:
I figure the reader will see the `generate-a3p-submission.sh` that's in HEAD. It will continue to evolve.